lib/kontrast/configuration.rb

```diff
@@ -71,11 +71,12 @@ def validate
             end
         end
 
-        def pages(width)
+        def pages(width, url_params = {})
```

Comment:
Good point, as long as there's only one second argument, in that case `url_params`, I'm totally fine with using a default value. definitely easier to read.